### ISSUE TITLE
emoji-mart css issue

### ISSUE THREAD

**Issue by fearless23** [GetStream/stream-chat-react#1061]
When compiling stream-chat-react with NextJS Version 10, get following error
`./node_modules/stream-chat-react/dist/css/index.css
Error: Can't resolve '../../node_modules/emoji-mart/css/emoji-mart.css' in 'C:\LOVED\lw-frontend-website\node_modules\stream-chat-react\dist\css'
    at runMicrotasks (<anonymous>)
`

stream-chat-react version = 6.4.3, also same error for version = 6.4.1

**Comment by DanC5** [comment]
@fearless23 I'm noticing the same issue as well and am working on a fix right now. We should have something to push out soon, thanks

**Comment by DanC5** [comment]
@fearless23 hopefully the CSS bundling issues are now fixed in the latest release, https://github.com/GetStream/stream-chat-react/releases/tag/v6.4.4. Let me know if this fixes the issue for you, thanks.

**Comment by fearless23** [comment]
@DanC5 The issue is fixed in 6.4.4. Thanks for quick fix.

**Comment by lifeinafolder** [comment]
As per the documentation [here](https://pubnub.github.io/react-chat-components/docs/?path=/docs/introduction-emoji-pickers--page), you have to `import` the package like this:

```
import { Picker } from "emoji-mart";
import "emoji-mart/css/emoji-mart.css";
```

but even when I do that, I get the following error:

![Screen Shot 2022-08-16 at 4 37 07 PM](https://user-images.githubusercontent.com/91226/185003161-7e5f13b4-9a86-40bf-938e-5a9341e30a38.png)

Looks like an issue with CSS bundling of dependencies.
